roles/rudder_agent/tasks/configure.yml

```diff
@@ -5,3 +5,21 @@
     owner: root
     group: root
     mode: '0644'
+
+- name: Set custom Rudder ports for 6 (min) version
+  replace:
+    path: "/var/rudder/cfengine-community/policy_server.dat"
+    regexp: "^x=rudder.example.com:(.*)$"
+    replace: "{{ rudder_policy_distribution_cfengine_port }}"
+  when: "{{ agent_version is version('6', operator='lt', strict=False) }}"
+
+- name: Set custom Rudder ports for 7.0 (min) version
+  replace:
+    path: "{{ item.path }}"
+    regexp: "{{ item.regex }}"
+    replace: "{{ item.replace }}"
+  with_items:
+    - { path: "/etc/rudder/etc/agent.conf", regex: "^https_port=(.*)$", replace: "{{ rudder_policy_distribution_https_port }}" },
```

Comment:
does this create the file and add the line if not present?

Reply:
this function only allows to check and replace a value in a file, I'm not sure which one creates the file if it doesn't exist

Reply:
We need to create the file as it does not exist by default

Reply:
it doesn't seem to be in conflict, in any case, no problem during my tests.

Reply:
Ok, I fixup the pr with the file creation!